apps/judicial-system/web/src/routes/Prosecutor/Indictments/Indictment/lib/getIncidentDescription.spec.ts

```diff
@@ -131,15 +140,16 @@ describe('getIncidentDescription', () => {
           IndictmentSubtype.TRAFFIC_VIOLATION,
         ],
       },
+      Gender.MALE,
+      {},
```

Comment:
Why are we changing the destructed object parameter definition? It makes it obvious for example that this is a crimescene object

Reply:
My background in compilers makes me want to do this. The previous implementation resulted in the creation of an object, which is then sent to the receiving method. The new implementation avoids this object creation, thus saving time and memory. Probably not big savings, but things add up.

The object types is obvoius in the method definition and any good idea should show you the type if you hover above the method you are calling.